packages/good-design/src/apps/ubi/screens/PostClaim.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useContext, useEffect } from "react";
+import { Center, HStack, Spinner, Text, VStack } from "native-base";
+import { isEmpty } from "lodash";
+import { NewsFeedContext, useTimer } from "@gooddollar/web3sdk-v2";
+
+import { NewsFeed } from "../../newsfeed";
+import { Image } from "../../../core/images";
+import { TransText, TransTitle } from "../../../core/layout";
+import { TransactionList } from "../components/TransactionStateCard";
+import { useClaimContext } from "../context";
+import { GoodButton } from "../../../core/buttons";
+
+import BillyHearts from "../../../assets/gifs/billy-hearts.gif";
+import SwitchArrows from "../../../assets/svg/switch-arrows.svg";
+
+const NextClaim = ({ time }: { time: Date }) => {
+  const [nextClaim, , setClaimTime] = useTimer(time);
+
+  useEffect(() => setClaimTime(time), [time.toString()]);
+
+  return (
+    <VStack alignItems="center">
+      <Image source={BillyHearts} alt="billy-hearts" width="100" height="120" />
+      <VStack space={0} backgroundColor="goodGrey.400" borderRadius="100" w="160" h="160" justifyContent="center">
+        <TransText t={/*i18n*/ "Your Next Claim"} textAlign="center" variant="sm-white-normal" />
+        <Text textAlign="center" fontFamily="heading" fontWeight="700" fontSize="l" color="white">
+          {nextClaim}
+        </Text>
+      </VStack>
+    </VStack>
+  );
+};
+
+export const PostClaim: FC = () => {
+  const { claimPools, claimDetails, claimedAlt, onNews, onTxDetails, switchChain } = useClaimContext();
+  const { feed } = useContext(NewsFeedContext);
+  const { transactionList } = claimPools ?? {};
+  const { claimTime, isWhitelisted } = claimDetails;
```

Comment:
what if pools have different claim times? how is claim time determinted?

Reply:
We currently base claim time solely on the main ubi pool. if we want to show timers for all pools separately we have to discuss with @decentralauren for a potential design change

Reply:
ok, i think it should simply be the nearest next pool claim time

Reply:
@sirpy @L03TJ3 I agree with hadar, it should be the nearest next pool claim time THAT person is eligible for (IE they would not get notice to "claim in 6 hours" a pool that they aren't a part of if the regular UBI claim time is 12 hours from now).